.gitignore

```diff
@@ -11,4 +11,7 @@ dist-types
 
 .ds_store
 
-.antlr
\ No newline at end of file
+.antlr
+
+# Intellij IDES add weird token files that we don't use
+**/grammar/*.tokens
```

Comment:
IDK how to fix it and have the antlr syntax highlighting, decided to just ignore this file